src/common/src/catalog/mod.rs

```diff
@@ -372,6 +373,25 @@ impl From<UserId> for u32 {
 #[cfg_attr(test, derive(PartialEq))]
 pub enum ConflictBehavior {
     NoCheck,
-    OverWrite,
+    Overwrite,
     IgnoreConflict,
 }
+
+impl ConflictBehavior {
+    pub fn from_protobuf(tb_conflict_behavior: &PbHandleConflictBehavior) -> Self {
+        match tb_conflict_behavior {
+            PbHandleConflictBehavior::Overwrite => ConflictBehavior::Overwrite,
+            PbHandleConflictBehavior::Ignore => ConflictBehavior::IgnoreConflict,
+            PbHandleConflictBehavior::NoCheck => ConflictBehavior::NoCheck,
+            _ => unreachable!(),
```

Comment:
By also treating `Unspecified` as `NoCheck` here, I guess this PR will not introduce breaking change any more.

Reply:
This PR also removes some `HandleConflictBehavior` messages in the proto. Will this introduce breaking change?